test/BlockChainHelper.cpp

```diff
@@ -462,13 +462,13 @@ vector<TestBlock> TestBlockChain::syncUncles(vector<TestBlock> const& uncles)
 	return validUncles;
 }
 
-TestTransaction TestTransaction::getDefaultTransaction()
+TestTransaction TestTransaction::getDefaultTransaction(string const& _nonce, string const& _gasLimit, string const& _data)
```

Comment:
Although you need a string in the function, those values are clearly numbers. Use u256, u64, bytes ...  and convert to string in the function.